packages/vue/src/view/layout/Footer.vue

```diff
@@ -0,0 +1,10 @@
+<template>
+  <footer>
+    <div class="container">
+      <a href="/" class="logo-font">conduit</a>
```

Comment:
Rlease use vue-router https://router.vuejs.org/guide/

Reply:
I didn't implement the pages/links yet, so I'll be moving to use the router component when I get there!

Reply:
Ok